R/plotPrevalence.R

```diff
@@ -38,6 +38,11 @@
 #'   \code{by} argument in 
 #'   \code{\link[scater:retrieveFeatureInfo]{?retrieveFeatureInfo}} for 
 #'   possible values. Only used with \code{layout = "point"}.
+#'   
+#' @param facet_by Specification of a feature to create facets.
+#' Argument can only be used in function getPrevalentAbundance. Value of argument
+#' must be in taxonomicRanks of \code{x}.
```

Comment:
That the arguments is only usable for getPrevalentAbundance should be visible from the man page. In addition I think you meant `plotPrevalentAbundace`

Shorter text: "taxonomic rank to facet the plot by. Value must be of \code{taxonomyRanks(x)}"

Reply:
Changed